.github/workflows/terraform_provider.yml

```diff
@@ -81,10 +81,28 @@ jobs:
       with:
         egress-policy: audit
 
-    - uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
-    - uses: bflad/tfproviderlint-github-action@3d16e28385eb2783700821cdf1b70f9f318bed3d # master
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+
+    - name: Set up Go
+      uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
       with:
-        args: ./...
+        cache: true
+        go-version-file: 'go.mod'
+      id: go
```

Comment:
a white-space could make it easier to understand which task have the `id: go` field